func_adl_xAOD/common/math_utils.py

```diff
@@ -36,7 +36,3 @@ def DeltaRAst(call_node):
 def DeltaR(eta1, phi1, eta2, phi2):
     'Calculate the DeltaR between two eta,phi specified vectors'
     raise NotImplementedError('DeltaR should never be called in python!')
-
-
-# Mark the DeltaR function as one that can be called.
-cpp_ast.method_names['DeltaR'] = DeltaRAst
```

Comment:
We want the definition here close to where the actual method is written - so in the same file, inside a registration function. Otherwise I'm worried when I come back to this in several years that it will take forever to realize after adding a new function here I have to also update another (perhaps several) files.